Upgrade all upgradable deps

nikomakela opened the pull request:
HH-372 TH-1376 LIIKUNTA-704.

Upgrade:
- date-fns
- date-fns-tz
- msw

Remove:
- domain (package dependency, since it was not used but reported as a vulnerability by Dependabot).

Fix: 
- lint-staged config for Components package

nikomakela committed:
chore(deps): upgrade dependencies

HH-372 TH-1376 LIIKUNTA-704.

Upgrade:
- date-fns
- date-fns-tz
- msw

nikomakela committed:
chore: fix lint-staged config for components package

HH-372 TH-1376 LIIKUNTA-704.

nikomakela committed:
deps(apps): remove "domain"

HH-372 TH-1376 LIIKUNTA-704.

[Prototype pollution in
dotty](https://github.com/City-of-Helsinki/events-helsinki-monorepo/security/dependabot/8)

"Domain" package removed, because it is not in any use, but it was
installed in all the apps.

terovirtanen commented:
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr792.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://liikunta-pr792.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://harrastukset-pr792.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr792.dev.hel.ninja** :rocket::rocket::rocket:

nikomakela committed:
fix(sports): errenous import statement to useCombinedSearchContext

HH-372 TH-1376 LIIKUNTA-704.

`yarn check:install` started to warn about import statement where a
shortcut-path was used. It was fine earlier, since there was a package
called "domain" installed.

terovirtanen commented:
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr792.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr792.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://tapahtumat-pr792.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://liikunta-pr792.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr792.dev.hel.ninja** :rocket::rocket::rocket:
